### PR TITLE
removed_the_bug in auto_issue_assignment

### PR DESCRIPTION
**Pull request by mihirm21**
## This PR

- remove the bug #77 

### Follow-up Tasks

https://github.com/marketplace/actions/auto-assign-issue 
According to this, need to add a personal access token with `read:org` scope instead of GITHUB_TOKEN on lines `43,53` for using `teams` in the referenced github action.

### How to test

Can be tested by editing any issue

### CODE REVIEW

**Comment by mihirm21** [comment]
@beeme1mr

File: .github/workflows/proposal_autoassign.yml [hunk 1]
```diff
@@ -1,40 +1,58 @@
...
+                  sdk-dotnet-maintainers
+                  sdk-golang-maintainers
+                  sdk-java-maintainers
+                  sdk-javascript-maintainers
+                  sdk-python-maintainers
+                  sdk-ruby-maintainers
+                  sdk-rust-maintainers
+                  sdk-php-maintainers
+                numOfAssignee: 10
+                removePreviousAssignees: true
```
Comment (beeme1mr) [inline .github/workflows/proposal_autoassign.yml:41]:
```suggestion
                removePreviousAssignees: false
```

File: .github/workflows/proposal_autoassign.yml [hunk 2]
```diff
@@ -1,40 +1,58 @@
...
+          - if: ${{steps.check_proposal_label.outputs.has_proposal_label == 'true'}}
+            name: Run auto-assignment script
+            id: check_proposal_label_1
+            run: |
+                category3=$(echo "${{ github.event.issue.body }}" | grep -e "Category" -A2 | grep -v "Category" | grep "\S" | tr -d " \t\n\r" | tr -d '"')
+                echo "category=$category3" >> "$GITHUB_OUTPUT"
+            shell: bash
+
+          - if: ${{ (steps.check_proposal_label_1.outputs.category == 'SDKs') || (steps.check_proposal_label_1.outputs.category == 'Specification')}}
+            name: 'Auto-assign issue1'
```
Comment (beeme1mr) [inline .github/workflows/proposal_autoassign.yml:28]:
```suggestion
            name: 'Auto-assign Spec and SDK Maintainers'
```

File: .github/workflows/proposal_autoassign.yml [hunk 3]
```diff
@@ -1,40 +1,58 @@
...
+                  sdk-ruby-maintainers
+                  sdk-rust-maintainers
+                  sdk-php-maintainers
+                numOfAssignee: 10
+                removePreviousAssignees: true
+                allowSelfAssign: true
+                repo-token: ${{ secrets.GITHUB_TOKEN }}
+          
+          - if: ${{ (steps.check_proposal_label_1.outputs.category == 'Flagd') || (steps.check_proposal_label_1.outputs.category == 'OpenFeatureOperator')}}
+            name: 'Auto-assign issue2'
```
Comment (beeme1mr) [inline .github/workflows/proposal_autoassign.yml:46]:
```suggestion
            name: 'Auto-assign Cloud Native Maintainers'
```

File: .github/workflows/proposal_autoassign.yml [hunk 4]
```diff
@@ -1,40 +1,58 @@
...
+            uses: pozil/auto-assign-issue@v1
+            with:
+                teams: cloud-native-maintainers
+                numOfAssignee: 5
+                removePreviousAssignees: true
```
Comment (beeme1mr) [inline .github/workflows/proposal_autoassign.yml:51]:
```suggestion
                removePreviousAssignees: false
```

**Review by beeme1mr — APPROVED** [review]

**Pull request merged**